### PR TITLE
Move all convenience scripts to the template

### PR DESCRIPTION
**Pull request by coorasse**
I keep moving stuff, it's now the turn of the other two convenience scripts.
With this comes also a small restructure of the first part of the guide.

### CODE REVIEW

File: ruby_on_rails/app_initialisation.md [hunk 1]
```diff
@@ -28,12 +28,6 @@ where the `project-name` is exactly the one you chose before.
 > If you need to do fancy stuff, discuss with your team the opportunity of including a js-bundling and css-bundling tool.\
 > We want to go ["no build"](https://www.youtube.com/watch?v=iqXjGiQ_D-A) whenever possible.
 
-> ⭐The Gemfile reads the required ruby version from the `.ruby-version` file.
-> [This is used by Heroku to determine what version to use.](https://devcenter.heroku.com/articles/ruby-versions)
-
-> ⭐️renuocop replaces the default rubocop-rails-omakase. We have our own set of rules at Renuo.
-> You can discuss them at https://github.com/renuo/renuocop and you can also contribute to them.
-
```
Comment (coorasse) [inline ruby_on_rails/app_initialisation.md:36]: moved down

File: ruby_on_rails/template.rb
Comment (coorasse) [inline ruby_on_rails/template.rb:1]: I'd like to check the opportunity of extracting those to files, when the template is finished.

File: ruby_on_rails/app_initialisation.md [hunk 2]
```diff
@@ -44,33 +38,33 @@ where the `project-name` is exactly the one you chose before.
...
-* Make the new scripts executable
+### Manual adjustments
 
-  ```sh
-  chmod +x bin/run bin/check
-  ```
+Please perform these adjustments manually:
 
-### ENV variables with Figaro
+#### ENV variables with Figaro
```
Comment (simon-isler) [inline ruby_on_rails/app_initialisation.md:67]: to be discussed in wg-engineering :)

**Review by simon-isler — APPROVED** [review]

File: ruby_on_rails/template.rb
Comment (simon-isler) [reply ruby_on_rails/template.rb:1]: yes this file can be refactored for sure

**Pull request closed without merging**